lib/fallbacks.json

```diff
@@ -0,0 +1,279 @@
+    "kbp": "fr",
+    "khw": "ur",
+    "kiu": "tr",
+    "kk": "kk-cyrl",
+    "kk-arab": "kk-cyrl",
+    "kk-cn": [
+        "kk-arab",
+        "kk-cyrl"
+    ],
+    "kk-kz": "kk-cyrl",
+    "kk-latn": "kk-cyrl",
+    "kk-tr": [
+        "kk-latn",
+        "kk-cyrl"
+    ],
+    "kl": "da",
+    "ko-kp": "ko",
+    "koi": "ru",
+    "krc": "ru",
+    "krl": "fi",
+    "ks": "ks-arab",
+    "ksh": "de",
+    "ku": "ku-latn",
+    "ku-arab": "ckb",
+    "kum": "ru",
+    "kv": "ru",
+    "lad": "es",
+    "lb": "de",
+    "lbe": "ru",
+    "lez": "ru",
+    "li": "nl",
+    "lij": "it",
+    "liv": "et",
+    "lki": "fa",
+    "lmo": "it",
+    "ln": "fr",
+    "lrc": "fa",
+    "ltg": "lv",
+    "luz": "fa",
+    "lzh": "zh-hant",
+    "lzz": "tr",
+    "mai": "hi",
+    "map-bms": [
+        "jv",
+        "id"
+    ],
+    "mdf": "ru",
+    "mg": "fr",
+    "mhr": "ru",
+    "min": "id",
+    "mo": "ro",
+    "mrj": "ru",
+    "mwl": "pt",
+    "myv": "ru",
+    "mzn": "fa",
+    "nah": "es",
+    "nan": [
+        "cdo",
+        "zh-hant"
+    ],
+    "nap": "it",
+    "nb": "nn",
+    "nds": "de",
+    "nds-nl": "nl",
+    "nl-informal": "nl",
+    "nn": "nb",
+    "nrm": "fr",
+    "oc": [
+        "ca",
+        "fr"
+    ],
+    "olo": "fi",
+    "os": "ru",
+    "pcd": "fr",
+    "pdc": "de",
+    "pdt": "de",
+    "pfl": "de",
+    "pms": "it",
+    "pnt": "el",
+    "pt": "pt-br",
+    "pt-br": "pt",
+    "qu": "es",
+    "qug": [
+        "qu",
+        "es"
+    ],
+    "rgn": "it",
+    "rmy": "ro",
+    "roa-tara": "it",
+    "rue": [
+        "uk",
+        "ru"
+    ],
+    "rup": "ro",
+    "ruq": [
+        "ruq-latn",
+        "ro"
+    ],
+    "ruq-cyrl": "mk",
+    "ruq-latn": "ro",
+    "sa": "hi",
+    "sah": "ru",
+    "scn": "it",
+    "sdh": "fa",
+    "ses": "fr",
+    "sg": "fr",
+    "sgs": "lt",
+    "sh": [
+        "bs",
+        "sr-el",
+        "hr"
+    ],
+    "sk": "cs",
+    "skr": "skr-arab",
+    "skr-arab": [
+        "ur",
+        "pnb"
+    ],
+    "sli": "de",
+    "sr": "sr-ec",
+    "srn": "nl",
+    "stq": "de",
+    "sty": "ru",
+    "su": "id",
+    "szl": "pl",
+    "tay": "zh-hant",
+    "tcy": "kn",
+    "tg": "tg-cyrl",
+    "tt": [
+        "tt-cyrl",
+        "ru"
+    ],
+    "tt-cyrl": "ru",
+    "ty": "fr",
+    "tyv": "ru",
+    "udm": "ru",
+    "ug": "ug-arab",
+    "vec": "it",
+    "vep": "et",
+    "vls": "nl",
+    "vmf": "de",
+    "vot": "fi",
+    "vro": "et",
+    "wa": "fr",
+    "wo": "fr",
+    "wuu": "zh-hans",
+    "xal": "ru",
+    "xmf": "ka",
+    "yi": "he",
+    "za": "zh-hans",
+    "zea": "nl",
+    "zh": "zh-hans",
+    "zh-cn": "zh-hans",
+    "zh-hant": "zh-hans",
+    "zh-hk": [
+        "zh-hant",
+        "zh-hans"
+    ],
+    "zh-mo": [
+        "zh-hk",
+        "zh-hant",
+        "zh-hans"
+    ],
+    "zh-my": [
+        "zh-sg",
+        "zh-hans"
+    ],
+    "zh-sg": "zh-hans",
+    "zh-tw": [
+        "zh-hant",
+        "zh-hans"
+    ]
+}
```

Comment:
new line - i have my editor configured to always add it

Reply:
Thanks, I thought mine did that too, but apparently not...